Documentation on custom activity (#310)
Unity-Technologies/com.unity.mobile.notifications: com.unity.mobile.notifications/Documentation~/FAQ.md

Some devices utilize [aggressive battery saver techniques](https://stackoverflow.com/questions/47145722/how-to-deal-with-huaweis-and-xiaomis-battery-optimizations) which restrict app background activities, unless the app has been whitelisted by the user in device settings. This behavior is observed on devices of certain manufacturers such as Asus, Huawei (including Honor), and Xiaomi. Adjusting battery settings to reduce power consumption might affect more devices.
This means that scheduled notifications will not be delivered if the app is closed or not running in the background. Currently, there's no available workaround besides encouraging the user to whitelist your app.

> #### Why is exception thrown on Android when trying to use notifications and notifications don't work?

Reviewer: ```suggestion
#### Why is an exception thrown on Android causing notifications to not work as expected?
```

> On Android notifications are setup to open activity when tapped. During initialization Unity determines the activity the same way as it does when notification arrives. If activity could not be determined (usually due to multiple choices), Unity throws exception in inform, that activity has to be explicitly configured in [notification settings](Settings.md#custom-activity).

Reviewer: ```suggestion
On Android, the notifications are set up to open an activity when tapped. During initialization, Unity determines the activity in the same way as it does when the notification is received. If Unity can not determine an activity (usually due to multiple choices), it throws an exception to indicate that the activity must be explicitly configured in [notification settings](Settings.md#custom-activity).
```

Reviewer: what is meant by "usually due to multiple choices"

Author: It means that Unity has found more than one suitable activity and has no way of knowing which one to use. We give UnityPlayerActivity and UnityPlayerGameActivity a priority, but it is possible that we find both of them or neither (in the later case we would use the only activity present in game, if there could be more than one).

> On Android, the notifications are set up to open an activity when tapped. During initialization, Unity determines the activity in the same way as it does when the notification is received. If Unity can not determine an activity (usually due to multiple choices), it throws an exception to indicate that the activity must be explicitly configured in the [notification settings](Settings.md#custom-activity).

Reviewer: ```suggestion
On Android, the notifications are set up to open an activity when tapped. During initialization, Unity determines the activity in the same way as it does when the notification is received. If Unity can not determine an activity (when more than one activity is active), it throws an exception to indicate that the activity must be explicitly configured in the [notification settings](Settings.md#custom-activity).
```
